hack/make-rules/test-e2e-node.sh

```diff
@@ -32,6 +32,7 @@ remote=${REMOTE:-"false"}
 images=${IMAGES:-""}
 hosts=${HOSTS:-""}
 metadata=${INSTANCE_METADATA:-""}
+gubernator=${GBR:-"false"}
```

Comment:
Document the flag in the makefile, here: https://github.com/kubernetes/kubernetes/blob/019b4c0e8f54c8998728d48dd157a5b865e011ed/Makefile#L138

Also, I think the convention is to spell out the flags fully, so it should be `GUBERNATOR`.